Erlang.gitignore

```diff
@@ -8,3 +8,6 @@ ebin/*.beam
 rel/example_project
 .concrete/DEV_MODE
 .rebar
+.rebar3
```

Comment:
The only suggestion I have here is to potentially tidy up this area of the template to make clear what rules are relevant to which versions of rebar (the [guide](https://www.rebar3.org/docs/from-rebar-2x-to-rebar3) provides some helpful context):

```gitignore
# rebar 2.x
...

# rebar 3
...
```

Is that something you're interested in adding while we're in here?

Reply:
I have added the headers according to what you said and moved some things to put them into the `rebar2.x` section.